packages/nimble-components/src/mapping/base/index.ts

```diff
@@ -0,0 +1,14 @@
+import { attr } from '@microsoft/fast-element';
+import { FoundationElement } from '@microsoft/fast-foundation';
+import type { MappingKey } from './types';
+
+/**
+ * Base class for mapping configuration elements
+ */
+export abstract class Mapping extends FoundationElement {
+    @attr({ mode: 'fromView' })
```

Comment:
As this is a departure from how we normally declare attributes (i.e. using the `fromView` mode), do you think it would make sense to provide a comment as to the reasoning behind using that?

Reply:
Now I'm not sure what this really does for us, so I'm removing it.

Reply:
All it's going to do is have it so that a programmatic change to this attribute isn't reflected back to the DOM. It's unclear to me if that would provide us any benefit.